cluster/juju/util.sh

```diff
@@ -61,7 +61,10 @@ function detect-nodes() {
     # ]
 
     # Strip out the IP addresses
-    export KUBE_NODE_IP_ADDRESSES=($(${JUJU_PATH}/return-node-ips.py "${ipoutput}"))
+    while IFS=$'\n' read -r ip;
+        do KUBE_NODE_IP_ADDRESSES+=("$ip");
+    done < <("${JUJU_PATH}/return-node-ips.py" "${ipoutput}")
```

Comment:
What shellcheck failure are you addressing?

Reply:
```
export KUBE_NODE_IP_ADDRESSES=($(${JUJU_PATH}/return-node-ips.py "${ipoutput}"))
                               ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
```

Reply:
reference: https://www.shellcheck.net/wiki/SC2207

Reply:
+1 for this change, thanks.